packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -515,7 +515,7 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
                         {errors.length}
                       </EuiBadge>
                     )}
-                    <CodeEditor
+                    <CodeEditor // TODO Unsure how to proceed, theme would most likely be provided by all callers, need CodeOwners help
```

Comment:
This seems to work fine now!

<img width="599" alt="image" src="https://github.com/elastic/kibana/assets/17003240/4765f642-4bf6-40fe-b3ad-ef5e5cd29e77">